integration-tests/mockApis/auth.js

```diff
@@ -4,7 +4,7 @@ const { stubFor, getRequests } = require('./wiremock')
 const createToken = () => {
   const payload = {
     sub: 'ITAG_USER',
-    user_name: 'ITAG_USER_USE_SUB',
+    name: 'SARAH ITAG',
```

Comment:
You've changed the name of the key here, was this originally an error with the test data or are we renaming it?

Reply:
the Auth Team have dropped user_name from the JWT in favour of Sub and I've asked them to add name. two unrelated things but that's what a JWT looks like now.